app/inpututils.cpp

```diff
@@ -1088,6 +1088,11 @@ bool InputUtils::isPointLayer( QgsVectorLayer *layer )
   return geometryFromLayer( layer ) == "point";
 }
 
+bool InputUtils::isMultiPointLayer( QgsVectorLayer *layer )
```

Comment:
maybe better to have more generic `isMultiPart()` function

Reply:
then in QML we can use `__inputUtils.isPointLayer( __activeLayer.vectorLayer ) && !__inputUtils.isMultiPartLayer( __activeLayer.vectorLayer )`